chains/substrate/events.go

```diff
@@ -60,17 +60,115 @@ type EventGenericTransfer struct {
+type EventMultisigExecuted struct {
+	Phase     types.Phase
+	Who       types.AccountID
+	TimePoint TimePoint
+	ID        types.AccountID
+	Result    types.DispatchResult
+	Topics    []types.Hash
+}
+
+// EventMultisigCancelled is emitted when a multisig operation has been cancelled by someone.
+// First param is the account that is approving, third is the multisig account.
+type EventMultisigCancelled struct {
+	Phase     types.Phase
+	Who       types.AccountID
+	TimePoint TimePoint
+	ID        types.AccountID
+	Topics    []types.Hash
+}
+
 type EventCodeUpdated struct {
 	Phase  types.Phase
 	Topics []types.Hash
 }
 
 type Events struct {
 	utils.Events
-	Bridge_FungibleTransfer    []EventFungibleTransfer    //nolint:stylecheck,golint
-	Bridge_NonFungibleTransfer []EventNonFungibleTransfer //nolint:stylecheck,golint
-	Bridge_GenericTransfer     []EventGenericTransfer     //nolint:stylecheck,golint
-	System_CodeUpdated         []EventCodeUpdated         //nolint:stylecheck,golint
+	ChainBridge_FungibleTransfer     []EventFungibleTransfer               //nolint:stylecheck,golint
+	ChainBridge_NonFungibleTransfer  []EventNonFungibleTransfer            //nolint:stylecheck,golint
+	ChainBridge_GenericTransfer      []EventGenericTransfer                //nolint:stylecheck,golint
+	System_CodeUpdated               []EventCodeUpdated                    //nolint:stylecheck,golint
+	Nfts_DepositAsset                []EventNFTDeposited                   //nolint:stylecheck,golint
+	Council_Proposed                 []types.EventCollectiveProposed       //nolint:stylecheck,golint
+	Council_Voted                    []types.EventCollectiveProposed       //nolint:stylecheck,golint
+	Council_Approved                 []types.EventCollectiveApproved       //nolint:stylecheck,golint
+	Council_Disapproved              []types.EventCollectiveDisapproved    //nolint:stylecheck,golint
+	Council_Executed                 []types.EventCollectiveExecuted       //nolint:stylecheck,golint
+	Council_MemberExecuted           []types.EventCollectiveMemberExecuted //nolint:stylecheck,golint
+	Council_Closed                   []types.EventCollectiveClosed         //nolint:stylecheck,golint
+	Fees_FeeChanged                  []EventFeeChanged                     //nolint:stylecheck,golint
+	MultiAccount_NewMultiAccount     []EventNewMultiAccount                //nolint:stylecheck,golint
+	MultiAccount_MultiAccountUpdated []EventMultiAccountUpdated            //nolint:stylecheck,golint
+	MultiAccount_MultiAccountRemoved []EventMultiAccountRemoved            //nolint:stylecheck,golint
+	MultiAccount_NewMultisig         []EventNewMultisig                    //nolint:stylecheck,golint
+	MultiAccount_MultisigApproval    []EventMultisigApproval               //nolint:stylecheck,golint
+	MultiAccount_MultisigExecuted    []EventMultisigExecuted               //nolint:stylecheck,golint
+	MultiAccount_MultisigCancelled   []EventMultisigCancelled              //nolint:stylecheck,golint
```

Comment:
Could we keep this to just the `ChainBridge_*` and `System_CodeUpdated` and move the rest to utils? Ideally want this to be the events the system actually uses

Reply:
yep!